dev-python/pgspecial/pgspecial-1.11.5.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+inherit distutils-r1
+
+DESCRIPTION="Python implementation of postgres meta commands"
+HOMEPAGE="https://github.com/dbcli/pgspecial"
+SRC_URI="https://github.com/dbcli/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	>=dev-python/click-4.1[${PYTHON_USEDEP}]
+	>=dev-python/python-sqlparse-0.1.19[${PYTHON_USEDEP}]
+"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
Well, strictly speaking it's a BDEPEND with EAPI-7. You can remove DEPEND altogether since distutils-r1 handles dependency for setuptools for you.